examples/demo-stf/src/sov-cli/main.rs

```diff
@@ -1,436 +1,12 @@
-use anyhow::Context;
```

Comment:
Why sov-cli lives in demo-stf? Can it be its own generic crate?

Reply:
Yes, we'll move this to a separate crate as part of #552